app/sections/view-implementation-timescales/macro.test.js

```diff
@@ -0,0 +1,58 @@
+import request from 'supertest';
+import cheerio from 'cheerio';
+import { createTestHarness } from '../../testUtils/testHarness';
+import * as settingsContext from './settings';
+
+const macroWrapper = `{% from './sections/view-implementation-timescales/macro.njk' import viewImplementationTimescales %}
+                        {{ viewImplementationTimescales(params) }}`;
+
+describe('view-implementation-timescales', () => {
+  it('should render the title of the section', (done) => {
+    const context = settingsContext;
```

Comment:
you can remove this and just pass settingsContext straight into the `createTestHarness` function